services/obstap/templates/tap-ingress-anonymous.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ template "cadc-tap-postgres.fullname" . }}-anonymous
+  labels:
+    {{- include "cadc-tap-postgres.labels" . | nindent 4 }}
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    nginx.ingress.kubernetes.io/proxy-connect-timeout: "900"
+    nginx.ingress.kubernetes.io/proxy-send-timeout: "900"
+    nginx.ingress.kubernetes.io/proxy-read-timeout: "900"
+    nginx.ingress.kubernetes.io/rewrite-target: "/tap/$1"
+    nginx.ingress.kubernetes.io/proxy-redirect-from: "http://$host/tap/"
+    nginx.ingress.kubernetes.io/proxy-redirect-to: "https://$host/api/obstap/"
+    nginx.ingress.kubernetes.io/ssl-redirect: "true"
+    nginx.ingress.kubernetes.io/use-regex: "true"
+    {{- with .Values.ingress.anonymousAnnotations }}
+    {{ toYaml . | indent 4}}
+    {{- end }}
+spec:
+  rules:
+    - host: {{ required "global.host must be set" .Values.global.host | quote }}
+      http:
+        paths:
+          - path: "/api/obstap/(availability)"
+            pathType: "ImplementationSpecific"
+            backend:
+              service:
+                name: {{ template "cadc-tap-postgres.fullname" . }}
+                port:
+                  number: 80
+          - path: "/api/obstap/(capabilities)"
+            pathType: "ImplementationSpecific"
+            backend:
+              service:
+                name: {{ template "cadc-tap-postgres.fullname" . }}
+                port:
+                  number: 80
+          - path: "/api/obstap/(swagger-ui.*)"
+            pathType: "ImplementationSpecific"
+            backend:
+              service:
+                name: {{ template "cadc-tap-postgres.fullname" . }}
+                port:
+                  number: 80
```

Comment:
This is already a regex, so these can all be merged into one rule, such as for the tap service.

Reply:
Done.